aspnet/tutorials/first-mvc-app/search.rst

```diff
@@ -91,96 +86,103 @@ However, you can't expect users to modify the URL every time they want to search
+However, even if you add this ``[HttpPost]`` version of the ``Index`` method, there's a limitation in how this has all been implemented. Imagine that you want to bookmark a particular search or you want to send a link to friends that they can click in order to see the same filtered list of movies. Notice that the URL for the HTTP POST request is the same as the URL for the GET request (localhost:xxxxx/Movies/Index) -- there's no search information in the URL. The search string information is sent to the server as a `form field value <https://developer.mozilla.org/en-US/docs/Web/Guide/HTML/Forms/Sending_and_retrieving_form_data>`__. You can verify that with the `F12 Developer tools <https://dev.windows.com/en-us/microsoft-edge/platform/documentation/f12-devtools-guide/>`__ or the excellent `Fiddler tool <http://www.telerik.com/fiddler>`__. Start the `F12 tool <https://dev.windows.com/en-us/microsoft-edge/platform/documentation/f12-devtools-guide/>`__ :
 
-.. image:: search/_static/f12.png
 
-Double tap the **http://localhost:1235/Movies  HTTP POST 200** line and then tap **Request body**. 
+Tap the **http://localhost:xxx/Movies	HTTP POST 200** line and then tap **Body  > Request Body**. 
 
 .. image:: search/_static/f12_rb.png
 
-You can see the search parameter and :doc:`XSRF <../../security/anti-request-forgery>` token in the request body. Note, as mentioned in the previous tutorial, the `Form Tag Helper <http://www.davepaquette.com/archive/2015/05/18/mvc-6-form-tag-helper.aspx>`__ generates an :doc:`XSRF <../../security/anti-request-forgery>` anti-forgery token. We're not modifying data, so we don't need to validate the token in the controller method.
+You can see the search parameter and `XSRF <:doc:../../security/anti-request-forgery>`__ token in the request body. Note, as mentioned in the previous tutorial, the :doc:`Form Tag Helper </mvc/views/working-with-forms>` generates an `XSRF <:doc:../../security/anti-request-forgery>`__ anti-forgery token. We're not modifying data, so we don't need to validate the token in the controller method.
```

Comment:
> `XSRF <:doc:../../security/anti-request-forgery>`__

Malformed links. Make path absolute instead of relative: `` :doc:`XSRF <:/security/anti-request-forgery>` ``